docs/logs/getting-started-console/Program.cs

```diff
@@ -17,33 +17,38 @@
 using Microsoft.Extensions.Logging;
 using OpenTelemetry.Logs;
 
-namespace SourceGeneration;
-
-public class Program
+using var loggerFactory = LoggerFactory.Create(builder =>
 {
-    public static void Main()
+    builder.AddOpenTelemetry(options =>
```

Comment:
Do we want to demonstrate resources here?

Reply:
I think it's fine to not include `Resources` here. We could show that in customizing the SDK exmaple.

Reply:
its there -> https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/docs/logs/customizing-the-sdk#setresourcebuilder